### DISCUSSION TITLE
Basic OSS stuffs we need to do

### DISCUSSION THREAD

**Discussion by oskardotglobal** [winapps-org/winapps#7] [Ideas]
Here's some of the less exiting stuff we need to consider when working as a team on this

We need:
- Contributor guidelines
- maybe Code of Conduct?
- A place to distribute docs (for tested apps, guides and stuff)

Also, iirc the project is currently licensed under GPL-3, right? Do we need a CLA? Who is the license holder?

**Comment by fbartels** [comment]
A popular framework for a code of conduct is https://www.contributor-covenant.org/, [Github also gives some examples](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) when it comes to contributor guidelines.

As for the docs I think the popular choices are to use the Github wiki, which would give externals a relatively easy way to also contribute to docs, or to use a static site generator like [mkdocs](https://squidfunk.github.io/mkdocs-material/) and then publish the resulting page via a github action. GIthub pages can be per repository or per organisation.

**Comment by AkechiShiro** [comment]
@oskardotglobal @LDprg should we also have a room on Matrix or some kind of place for the winapps community and for users to interact together ?

I can take care of creating a Matrix Room, but what do you think of this ?

**Reply by LDprg** [reply]
A matrix room would be nice for a more direct way of communicating.

The only downside is that we split information on github discussion and matrix, but I guess there is nothing we can really do about this.

**Reply by oskardotglobal** [reply]
Sure, I've gone ahead and created one: https://matrix.to/#/%23winapps:oskar.global
